pallets/issue/src/lib.rs

```diff
@@ -674,4 +704,46 @@ impl<T: Config> Pallet<T> {
 	fn issue_minimum_transfer_amount(currency_id: CurrencyId<T>) -> Amount<T> {
 		Amount::new(IssueMinimumTransferAmount::<T>::get(), currency_id)
 	}
+
+	fn increase_interval_volume(issue_amount: Amount<T>) -> Result<(), DispatchError> {
+		Ok(if let Some(limit_volume) = LimitVolumeAmount::<T>::get() {
+			let issue_volume = Self::convert_into_limit_currency_id_amount(issue_amount)?;
+			let current_volume = CurrentVolumeAmount::<T>::get();
+			let new_volume = current_volume
+				.checked_add(&issue_volume.amount())
```

Comment:
We could use a `.saturating_add()` here because that should fit perfectly for our use case